package.json

```diff
@@ -52,6 +57,7 @@
   },
   "files": [
     "src/sap/fhir/**/*.js",
-    "ui5.yaml"
+    "ui5.yaml",
+    "LICENSE.txt"
```

Comment:
FYI, LICENSE files are always included: https://docs.npmjs.com/files/package.json#files

Reply:
Thx for this hint.